sympy/combinatorics/free_group.py

```diff
@@ -127,6 +127,7 @@ class FreeGroup(DefaultPrinting):
     """
     is_associative = True
     is_group = True
+    is_fp_group = True
```

Comment:
> This is how I think about it. .is_FpGroup would return True for an instance of FpGroup

I am -1 to this kind of functionality. I know this is abundantly done in SymPy codebase, but it is just not that useful. I think ``isinstance`` method covers all of the use cases. Plus, ``isinstance`` is always more clearer to me.